MantleTests/MTLTestModel.m

```diff
@@ -461,6 +462,49 @@ + (NSValueTransformer *)nestedRangeJSONTransformer {
 }
 @end
 
+@implementation MTLDotLiteralKeyPathModel
+
++ (NSDictionary *)JSONKeyPathsByPropertyKey {
+	return @{
+			 @"name": MTLKeyPath(@"literal.name"),
+			 @"nestedName": MTLKeyPath(@"nested", @"literal.name"),
+			 @"fullName": @[ MTLKeyPath(@"literal.first_name"), MTLKeyPath(@"literal.last_name") ],
+			 @"nestedFullName": @[ MTLKeyPath(@"nested", @"literal.first_name"), MTLKeyPath(@"nested", @"literal.last_name") ]
+			 };
+}
+
++ (NSValueTransformer *)fullNameJSONTransformer {
+	return [MTLValueTransformer transformerUsingForwardBlock:^id(id value, BOOL *success, NSError *__autoreleasing *error) {
+		NSString *firstName = value[MTLKeyPath(@"literal.first_name")];
```

Comment:
Is this right? Shouldn't we look up using the `NSString` `@"literal.first_name"` here instead of the `MTLKeyPath` containing the array `@[ @"literal.first_name" ]`?